src/System.Net.Security/tests/FunctionalTests/CertificateValidationRemoteServer.cs

```diff
@@ -12,7 +12,6 @@ namespace System.Net.Security.Tests
 {
     public class CertificateValidationRemoteServer
     {
-        [ActiveIssue(5555, PlatformID.Linux)]
```

Comment:
@bartonjs This only purpose of this test is to validate that it can successfully connect to a trusted server. We have other tests that validate the scenario below.